man/systemd.netdev.xml

```diff
@@ -1070,6 +1070,33 @@
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>ActiveSlave=</varname></term>
+        <listitem>
+          <para>A boolean. Specifies the new active slave. The <literal>ActiveSlave=</literal>
+          option is only valid for following modes.
```

Comment:
if this is followed by a list, then this should really be using a colon (":") instead of a full stop (".")...